.github/workflows/automerge.yml

```diff
@@ -0,0 +1,24 @@
+name: automerge
+on: [push, pull_request]
+
+jobs:
+  build:
```

Comment:
Could you add another job here for the unit tests called `test`, and make `automerge` require it? Or perhaps you can add a extra step in the `build` job. I'm not sure which of the two is better, so I'll leave that up to you.

If you do that, then I will merge this PR :)

Reply:
Done :D - I added `yarn test` right after build, which means if either `yarn build` or `yarn test` fail, the `automerge` step won't be triggered.

Reply:
Thanks you!